vendor-extra/LiberoPatternsBundle/src/Resources/views/reference.html.twig

```diff
@@ -0,0 +1,58 @@
+{%- import '@LiberoPatterns/html.html.twig' as html -%}
+
+{%- set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['reference']) }) -%}
+
+<article {{- html.attributes(attributes) }}>
+
+  {%- block body -%}
+
+    {%- with {attributes: {class: []}} -%}
+      {%- block header -%}
+
+        {%- with heading|merge({text: {text: heading.text} }) only -%}
+          {%- block heading -%}
+
+            {%- set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['reference__heading']) }) -%}
+            {%- set level = level|default(3) -%}
+
+            {%- include '@LiberoPatterns/heading.html.twig' -%}
+
+          {%- endblock heading -%}
+        {%- endwith -%}
+
+        {%- with authors|default({}) only -%}
+          {%- block authors -%}
+
+            {%- if items|default([])|length -%}
+              {%- include '@LiberoPatterns/inline-list.html.twig' with {small: true} -%}
+            {%- endif -%}
+
+          {%- endblock authors -%}
+        {%- endwith -%}
+
+        {%- with details|default({}) only -%}
+          {%- block details -%}
+
+            {%- if text is defined -%}
+              {%- include '@LiberoPatterns/paragraph.html.twig' -%}
```

Comment:
Not for now, but not sure of the best way to handle multiple pieces of information being brought together here.